tests/unit/utils.py

```diff
@@ -8,9 +8,13 @@ def get_app_dusty_schema(doc):
         doc['image'] = ''
     if 'repo' not in doc:
         doc['repo'] = ''
+    if 'mount' not in doc:
+        doc['mount'] = ''
     return DustySchema(app_schema, doc)
```

Comment:
I feel like this should die and our fixtures should be forced to be explicit about the shit they need. This seems overly magical.